components/sections/app-header.vue

```diff
@@ -1,9 +1,69 @@
 <template>
 	<header class="bg-rose-500 dark:bg-stone-900">
-		<div class="container px-5 py-4 mx-auto flex items-center sm:flex-row flex-col">
-			<p class="text-lg sm:py-2">
+		<div class="container px-5 py-4 mx-auto flex items-center flex-row">
+			<p class="text-lg sm:py-2 grow">
 				<NuxtLink href="/" class="font-extrabold text-2xl text-stone-100 hover:text-stone-200">Meme Generator</NuxtLink>
 			</p>
+			<nav>
+				<nuxt-link
+					v-if="!$auth.loggedIn.value"
+					class="bg-gray-200/70 hover:bg-gray-200/80 dark:bg-stone-800 hover:dark:bg-stone-700 p-3.5 rounded-xl"
+					:to="getLoginURL()"
+				>
+					Login
+				</nuxt-link>
+				<button
+					v-else
+					title="Logout"
+					aria-label="Logout"
+					@click="authLogout()"
+					class="flex items-center gap-2 bg-gray-200/70 hover:bg-gray-200/80 dark:bg-stone-800 hover:dark:bg-stone-700 px-3.5 py-2 rounded-xl"
+				>
+					{{ $auth.session.value?.name }}
+					<img v-if="isDefault" :src="defaultAvatar" alt="Default Avatar" class="rounded-full w-10" />
+					<picture v-else>
+						<source v-if="isAnimated" type="image/gif" :srcset="makeSrcset('gif')" />
+						<source type="image/webp" :srcset="makeSrcset('webp')" />
+						<source type="image/png" :srcset="makeSrcset('png')" />
+						<img :src="createUrl('png', 128)" alt="Avatar" class="rounded-full w-8" />
```

Comment:
automatically prefer *not* GIF when reduced-motion is chosen by the user https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Reply:
I called it, resolved in ebe5275.